src/plots/mapbox/layout_attributes.js

```diff
@@ -217,7 +223,15 @@ var attrs = module.exports = overrideAll({
                 dflt: 2,
                 role: 'style',
                 description: [
-                    'Sets the line width.',
+                    'Sets the line width (mapbox.layer.paint.line-width).',
+                    'Has an effect only when `type` is set to *line*.'
+                ].join(' ')
+            },
+            dash: {
+                valType: 'data_array',
```

Comment:
Does `mapbox.layer.paint.line-dasharray` also support scalars?

Reply:
"Optional array of numbers greater than or equal to 0" [here](https://docs.mapbox.com/mapbox-gl-js/style-spec/#layers-line)

Reply:
Thanks!